doc/OutlierFiltersFamilies.md

```diff
@@ -0,0 +1,377 @@
+| [Tutorials Home](index.md) | [Previous](ApplyingDataFilters.md) | [Next](ICPIntro.md) |
+| :--- | :---: | ---: |
+
+# Outlier Filters Families
+
+This section adresses the various filters and tries to organise them in families. The families are based on the influence functions of the cost function associated to an outlier filter. The purpose of outlier filtering, applied to point cloud registration, is to reduce the effect of outliers, improve the error minimisation and improve the crispness of the registered point clouds.
+
+Appart for the Other family, the presentation of the families and functions is the following. The functions were plotted with an error variable (noted e) ranging from [-4,4] with a step of 0.01. The plot of a family is made with a value of k = 1. The first plot of a function is made with values of k = 0.5, k = 2 and k = 4. Then a second plot is proposed for values of k ranging from [0,2] with a step of 0.1.
+
+This catalog was made from thanks to Philippe Babin's Mémoire on the analysis of error functions for the Iterative closest point algorithm: P.Babin, "Analysis of Error Functions for the Iterative Closest Point Algorithm", pp.70-72.
+
+Libpointmatcher provides an implementation of the following outlier filters:
+
+* Cauchy
+* Welsch
+* Switchable-Constaint
+* Geman-McClure
+* Tukey
+* Huber
+* L1
+* Student
+
+## Filter Index
+### [Pseudo-Sinus Family](#pseudosinfamilyhead)
+1. [Cauchy](#cauchyhead)
+2. [Geman-McClure (GM)](#gemanmcclurehead)
+3. [Welsch](#welschhead)
+4. [Tukey](#tukeyhead)
+5. [Andrew](#andrewhead)
+6. [Kernel-Correlation (KC)](#kernelcorrelationhead)
+7. [Fast Global Registration (FGR)](#fastglobalregistrationhead)
+
+### [Linear Family](#linearfamilyhead)
+8. [L2](#l2head)
+9. [Switchable-Constraint (SC)](#switchableconstrainthead)
+10. [Maximum-Distance (MaxDist)](#maximumdistancehead)
+11. [Generalized ICP (GICP)](#gicphead)
+
+### [Step Family](#stepfamilyhead)
+12. [L1](#l1head)
+13. [Huber](#huberhead)
+14. [Logistic](#logistic)
+15. [LM-ICP](#lmicphead)
```

Comment:
LM-ICP is just Huber, I don't think it should be included in the list outlier filter. It would preferable if in the section about Huber, it is mentioned that LM-ICP used it.

Reply:
Corrected